tests/test_datasets/test_dataset.py

```diff
@@ -219,6 +219,11 @@ def test_get_data_corrupt_pickle(self):
         self.assertIsInstance(xy, pd.DataFrame)
         self.assertEqual(xy.shape, (150, 5))
 
+    def test_equality_comparison(self):
+        self.assertTrue(self.iris == self.iris)
```

Comment:
Is there a reason to not go for `self.assertEqual(self.iris, self.iris)`?

Reply:
No, it was just the first thing that came to my mind. I changed it as you suggested.